plugins/spu2-x/src/Windows/ConfigAdvancedVolume.cpp

```diff
@@ -0,0 +1,209 @@
+            break;
+
+        case WM_HSCROLL: {
+            wmId = LOWORD(wParam);
+            HWND hwndDlg = (HWND)lParam;
+
+            int curpos = HIWORD(wParam);
+
+            switch (wmId) {
+                case TB_LINEUP:
+                case TB_LINEDOWN:
+                case TB_PAGEUP:
+                case TB_PAGEDOWN:
+                case TB_TOP:
+                case TB_BOTTOM:
+                    curpos = (int)SendMessage(hwndDlg, TBM_GETPOS, 0, 0);
+
+                case TB_THUMBPOSITION:
+                case TB_THUMBTRACK:
+                    Clampify(curpos,
+                             (int)SendMessage(hwndDlg, TBM_GETRANGEMIN, 0, 0),
+                             (int)SendMessage(hwndDlg, TBM_GETRANGEMAX, 0, 0));
+
+                    SendMessage((HWND)lParam, TBM_SETPOS, TRUE, curpos);
+
+                    if (hwndDlg == GetDlgItem(hWnd, IDC_VOLADJ_CDB)) {
+                        swprintf_s(temp, L"%d dB", curpos - 25);
+                        SetDlgItemText(hWnd, IDC_ADV_VOL_C_LABEL, temp);
+                    } else if (hwndDlg == GetDlgItem(hWnd, IDC_VOLADJ_FLDB)) {
+                        swprintf_s(temp, L"%d dB", curpos - 25);
+                        SetDlgItemText(hWnd, IDC_ADV_VOL_FL_LABEL, temp);
+                    } else if (hwndDlg == GetDlgItem(hWnd, IDC_VOLADJ_FRDB)) {
+                        swprintf_s(temp, L"%d dB", curpos - 25);
+                        SetDlgItemText(hWnd, IDC_ADV_VOL_FR_LABEL, temp);
+                    } else if (hwndDlg == GetDlgItem(hWnd, IDC_VOLADJ_BLDB)) {
+                        swprintf_s(temp, L"%d dB", curpos - 25);
+                        SetDlgItemText(hWnd, IDC_ADV_VOL_BL_LABEL, temp);
+                    } else if (hwndDlg == GetDlgItem(hWnd, IDC_VOLADJ_BRDB)) {
+                        swprintf_s(temp, L"%d dB", curpos - 25);
+                        SetDlgItemText(hWnd, IDC_ADV_VOL_BR_LABEL, temp);
+                    } else if (hwndDlg == GetDlgItem(hWnd, IDC_VOLADJ_SLDB)) {
+                        swprintf_s(temp, L"%d dB", curpos - 25);
+                        SetDlgItemText(hWnd, IDC_ADV_VOL_SL_LABEL, temp);
+                    } else if (hwndDlg == GetDlgItem(hWnd, IDC_VOLADJ_SRDB)) {
+                        swprintf_s(temp, L"%d dB", curpos - 25);
+                        SetDlgItemText(hWnd, IDC_ADV_VOL_SR_LABEL, temp);
+                    } else if (hwndDlg == GetDlgItem(hWnd, IDC_VOLADJ_LFEDB)) {
+                        swprintf_s(temp, L"%d dB", curpos - 25);
+                        SetDlgItemText(hWnd, IDC_ADV_VOL_LFE_LABEL, temp);
+                    }
+                    break;
+
+                default:
+                    return FALSE;
+            }
+        } break;
+
+        default:
+            return FALSE;
+    }
+    return TRUE;
+}
+
+void AdvancedVolumeConfig::OpenDialog()
+{
+    INT_PTR ret = DialogBoxParam(hInstance, MAKEINTRESOURCE(IDD_CONFIG_ADVANCEDVOLUME), GetActiveWindow(), (DLGPROC)DialogProc, 1);
```

Comment:
The 5th parameter seems unused. I think you could omit it and just use `DialogBox` instead.